refgenconf/populator.py

```diff
@@ -29,6 +29,43 @@ def looper_refgenie_populate(namespaces):
     ):
         rgc_path = namespaces["pipeline"]["var_templates"]["refgenie_config"]
         rgc = refgenconf.RefGenConf(rgc_path)
+
+        # Populate a dict with paths for the given sample's genome
+        # paths_dict = {}
+        # for a in rgc.list_assets_by_genome(g):
+        #     paths_dict[a] = rgc.seek(g, a, "default")
+
+        complete_seek_key_dict = rgc.list_seek_keys_values(
+            genomes=namespaces["sample"]["genome"]
```

Comment:
Is `genome` key always available there? 

If there was no `genome` key in the sample namespace it would result in an obscure `KeyError`

Reply:
ok fixed.